assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -193,10 +179,43 @@ const sendFormSubmitEventForPayPalRecurring = () =>
     onFormSubmit(formSubmitParameters);
   };
 
-function setValueAndTogglePayPal<T>(setStateValue: T => Action, value: T) {
-  return (dispatch: Function): void => {
+
+function enableOrDisableForm(formIsValid: boolean, state: State, dispatch: Function) {
+
+  const shouldEnable = checkoutFormShouldSubmit({
```

Comment:
maybe this function can be called `checkoutFormIsSubmittable` to match the new language